ocaml/idl/datamodel_pool.ml

```diff
@@ -1024,6 +1024,42 @@ let set_uefi_certificates =
       ]
     ~allowed_roles:_R_POOL_ADMIN ()
 
+let update_sync_frequency =
+  Enum
+    ( "update_sync_frequency"
+    , [
+        ("daily", "Indicates the pool update synchronization is scheduled daily")
+      ; ( "weekly"
+        , "Indicates the pool update synchronization is scheduled weekly"
+        )
+      ; ( "monthly"
+        , "Indicates the pool update synchronization is scheduled monthly"
+        )
+      ]
+    )
+
+let configure_update_sync =
+  call ~name:"configure_update_sync"
```

Comment:
I would try to simplify this: "The frequency at which information about new updates are downloaded:  daily, weekly, or monthly."

Reply:
I changed the description to: "The frequency at which updates are synced from remote CDN: daily, weekly, or monthly."